README.md

```diff
@@ -729,7 +729,7 @@ Use `!isNaN` in combination with `parseFloat()` to check if the argument is a nu
 Use `isFinite()` to check if the number is finite.
```

Comment:
We should update the description to make sure people understand what it is that the snippet does. Also make changes to the `validate-number.md` file, not README. Apart from that, we are solid! ;)